starboard/raspi/shared/test_filters.py

```diff
@@ -54,6 +54,13 @@
         # Filter failed tests.
         'PlayerComponentsTests/PlayerComponentsTest.*',
     ],
+    'net_unittests': [
+        'VersionIncludeStreamDependencySequence/' +
+        'QuicNetworkTransactionTest.NetworkIsolationTunnel*',
+        'SchemeHostPortMatcherRuleTest.' +
```

Comment:
Does the "+" work? Do you want to do `'SchemeHostPortMatcherRuleTest.*',` ?

Reply:
It should work? At least it works in Evergreen/modular builds. I haven't check the Raspi test result yet, since the run just started.

No, I only want to disable this test: 'SchemeHostPortMatcherRuleTest.SchemeHostPortMatcherHostnamePatternRule_PunnyCodeHostname'. And it's too long, so I break it with "+".

Reply:
I would recommend disabling pylint for these lines instead, as I often see filters being mistakenly changed when it's split across lines